pages/about.tsx

```diff
@@ -15,139 +15,43 @@ export default createPage({
-                    <a
-                      href="https://www.youtube.com/watch?v=XBFWFC2vtiY&t=770s"
-                      target="_blank"
-                      rel="noreferrer"
-                    >
-                      <Image
-                        src={faithDemocracy}
-                        className={style.posterImages}
-                        alt=""
-                      />
-                    </a>
-                  </div>
-                  <div className={style.posterImgLink} id="leftPoster">
-                    <a
-                      href="https://www.youtube.com/watch?v=XF2IGKopwdk"
-                      target="_blank"
-                      rel="noreferrer"
-                    >
-                      <Image
-                        src={democracyReform}
-                        className={style.posterImages}
-                        alt=""
-                      />
-                    </a>
-                  </div>
-                </div>
-
-                <div className={style.secondPosterCol}>
-                  <div className={style.posterImgLink} id="middlePoster1">
-                    <a
-                      href="https://www.youtube.com/watch?v=utCYTU3miOg&t=3s"
-                      target="_blank"
-                      rel="noreferrer"
-                    >
-                      <Image
-                        src={racialJustice}
-                        className={style.posterImages}
-                        alt=""
-                      />
-                    </a>
-                  </div>
-                  <div className={style.posterImgLink} id="middlePoster2">
-                    <a
-                      href="https://www.youtube.com/watch?v=DLjdC8BFlvY&t=34s"
-                      target="_blank"
-                      rel="noreferrer"
-                    >
-                      <Image
-                        src={privacyDemocracy}
-                        className={style.posterImages}
-                        alt=""
-                      />
-                    </a>
-                  </div>
-                </div>
-              </div>
-            </div>
+              events to galvanize and encourage faith in democracy.  Project co-founder Matt Victor is a JD student at BC and the founder of GGP.
+              </p>
+              <p>
+              The Berkman Klein Center's mission is to explore and understand cyberspace; to study its development, dynamics, norms, and standards; and to assess the need or lack thereof for laws and sanctions. Project co-founder <a href="https://cyber.harvard.edu/people/nathan-sanders">Nathan Sanders</a> is a past Fellow and current Affiliate at BKC.
+              </p>
+              <h1>Project History</h1>
+              <p>
+              </p>
+              <h1>Project Motivations</h1>
+              <p>
+              </p>
           </div>
 
           <Twitter />
```

Comment:
Does it make sense to keep the GGP twitter feed in this about page? If so we should file create an issue to make the about page responsive since this component causes the text to flow weirdly on mobile.

Reply:
Agreed.  It's also not great for the demo since the feed is months stale.  I'll comment it out for now and link this comment to a new issue.

FYI @mvictor55